cmd/osbuild-composer-dbjobqueue-tests/main_test.go

```diff
@@ -144,3 +148,8 @@ func testDeleteJobResult(t *testing.T, q *dbjobqueue.DBJobQueue) {
 	require.NoError(t, err)
 	require.Nil(t, r)
 }
+
+func testClose(t *testing.T, q *dbjobqueue.DBJobQueue) {
+	err := q.Close()
```

Comment:
`q.Close()` doesn't return any value, does it?

Reply:
Right, yeah. Hmmm, to test that I am going to check if a subsequent call returns an error. That's the only idea I have other than making something exported to check if the connection was closed.